website/docs/r/is_lb_listener.html.markdown

```diff
@@ -188,7 +188,7 @@ Review the argument references that you can specify for your resource.
       - Not already be the default_pool for another listener 
 - `certificate_instance` - (Optional, String) The CRN of the certificate instance, it is applicable(mandatory) only to https protocol.
 
```

Comment:
if its removed then remove optional ?

Reply:
This is still has to be to support the CRNs of secrets from secrets manager.